debugging/coreclr/dockerManager.ts

```diff
@@ -217,7 +217,7 @@ export class DefaultDockerManager implements DockerManager {
                         env: options.env,
                         envFiles: options.envFiles,
                         labels: options.labels,
-                        volumes
+                        volumes: [...volumes, ...options.volumes]
```

Comment:
Shouldn't the `options.externalHosts` and `options.ports` be passed as well?

Reply:
Yes, I'll fix this point too.